ts/src/autocomplete.ts

```diff
@@ -87,7 +87,8 @@ export class Autocomplete {
             let wrapperParent;
             if(editingArea.parentElement?.parentElement?.classList.contains("collapsible")) {
                 // 2.1.55+
-                wrapperParent = editingArea.parentElement.parentElement
+                wrapperParent = fieldElm.parentElement?.parentElement?.parentElement
+                wrapperParent.style.display = "block"
```

Comment:
This element has `display: contents`, which I found is making this harder to fix for some reason. I didn't notice any side effects caused by changing it.